README.rdoc

```diff
@@ -64,16 +64,30 @@ Please, have a look the wiki pages about development workflow and code standards
 
 * Services (job queues, cache servers, search engines, etc.)
 
-  Kalibro Processor
+  Kalibro Processor, Kalibro Configurations and Analizo are necessary to set up properly your environment.
+
+  The clones of Kalibro processor and Kalibro configurations must be in the same directory of the Prezento.
+  something like +mezuro/prezento+, +mezuro/kalibro_configurations+, +mezuro/kalibro_processor+.
+
+  <b>Kalibro Processor</b>
+  
   You can find it at https://github.com/mezuro/kalibro_processor
   By default Mezuro will expect it to be running on port 8082 at localhost.
   In order to run it as expected, run on two different terminal instances: ```RAILS_ENV=local rails s -p 8082```; and ```RAILS_ENV=local rake jobs:work```.
 
-  Kalibro Configurations
+  <b>Kalibro Configurations</b>
+  
   You can find it at https://github.com/mezuro/kalibro_configurations
   By default Mezuro will expect it to be running on port 8083 at localhost.
   In order to run it as expected, run on two different terminal instances: ```rails s -p 8083```.
 
+  <b>Analizo</b>
```

Comment:
This is not the right syntax for RDOC. See http://docs.seattlerb.org/rdoc/RDoc/Markup.html#class-RDoc::Markup-label-Text+Markup.